monkey/common/tags/attack.py

```diff
@@ -18,3 +18,6 @@
 CREDENTIALS_FROM_PASSWORD_STORES_T1555_TAG = "attack-t1555"  # credential access
 GATHER_VICTIM_HOST_INFORMATION_T1592_TAG = "attack-t1592"
 ACTIVE_SCANNING_T1595_TAG = "attack-t1595"
+DEFACEMENT_T1491_TAG = "attack-t1491"
+INTERNAL_DEFACEMENT_T1491_001_TAG = "attack-t1491.001"
+EXTERNAL_DEFACEMENT_T1491_002_TAG = "attack-t1491.002"
```

Comment:
If we're adding tags for sub-techniques, we should make a note to update the old ones too.